### DISCUSSION TITLE
VAD model❓ Questions / Help / Support

### DISCUSSION THREAD

**Discussion by changduckGO** [snakers4/silero-vad#146] [Q&A] [answered]
Is there a way to fine-tune using my own data in the Silero VAD model?
If can't fine-tune, is there any other way to improve the Silero VAD Model performance?

Thank you for reading!

**Comment by snakers4** [comment, marked as answer]
Hi,

As for fine-tuning, we do not support / provide code for it.

As for tuning the hyper-params, you may want to check this thread out - https://github.com/snakers4/silero-vad/discussions/141, it should take 5-10 mins of work per domain, usually it should be enough.

If you mean performance as "speed", we have reached a limit after which we do not know low-maintenance and easy ways to improve it.

**Reply by changduckGO** [reply]
Thank you so much for the speedy reply 😄😄
Have a nice day!!
